### ISSUE TITLE
[A11y] Work Show Page-Multiple - Checkbox labels

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1991]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n

Checkboxes next to items under "IS PART OF", "RELATED ITEMS", and "CONTAINED IN THIS OBJECT" have no label.

### Suggested Fix

I'm not sure - what are these checkboxes doing? I can't seem to find out what they're used for, so I'm not sure what the label should actually be.... It's also tough to come up with a useful label since the objects all have the same name visually.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Verify the checkboxes have some kind of label for assistive tech, and that the label is useful for the purpose the checkboxes serve.

**Comment by jechols** [comment]
I think this still has to be failed - there's a label on the checkbox, but it doesn't seem to tell me what they are doing. It says "Select X for saving", but it doesn't do anything after that selection, and there doesn't seem to be anything that allows saving checked items. This is a usability issue for sure, but it's also accessibility because those checkboxes imply functionality that just doesn't exist, but which a sighted user can easily ignore. A screen-reader user might go and check a bunch of items and then activate the first "save" only to find it silently unchecked everything but the one item nearest that "save" button.

**Comment by CGillen** [comment]
Fixed further by #2189
